WCON_format.md

```diff
@@ -280,12 +283,10 @@ An example WCON file with a complete metadata section is given below.
            "media":"NGM",
            "sex":"hermaphrodite",
            "stage":"dauer",
-           "age":38.4,
+           "age":18.511,
            "strain":"CB4856",
-           "protocol":[
-               "dauer induction by method in J. Doe, 'Get Dauers', J. of Stuff, v1 p234",
-               "worm transferred to arena 1-2 minutes before recording started"
-           ],
+           "protocol":
+               "dauer induction by method in J. Doe, 'Get Dauers', J. of Stuff, v1 p234\nworm transferred to arena 1-2 minutes before recording started",
```

Comment:
I wanted this not to contain weird formatting characters like `\n` to seem more accessible.  Also, the `protocol` field is stated to allow arrays.